src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -2,7 +2,6 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
-import matchSorter from 'match-sorter';
```

Comment:
Is this related to the ticket?

Reply:
This was part of the react table filters, so when I removed the table it was no longer required as it raises `unused import` warnings.